.github/ISSUE_TEMPLATE/01_bug_report.yml

```diff
@@ -0,0 +1,85 @@
+name: Bug Report
+description: 'Any bug about core nzbget, EXCLUDING extensions, performance issues or compiling nzbget.'
+labels: ['bug']
+body:
+- type: checkboxes
+  attributes:
+    label: Is there already an issue for your problem?
+    description: Please make sure you are not creating an already submitted <a href="https://github.com/nzbgetcom/nzbget/issues">Issue</a>. Check closed issues as well, because your issue may have already been fixed.
+    options:
+      - label: I have checked older issues, open and closed
+        required: true
+- type: dropdown
+  attributes:
+    label: NZBGet Version
+    description: Which version of NZBGet has this bug?
+    options:
+    - v23-stable
+    - v23-testing
+    - v22-stable (nzbgetcom takeover)
+    - v22-testing (nzbgetcom takeover)
+    - v21 or earlier (orignal nzbget)
+  validations:
+    required: true
+- type: dropdown
+  attributes:
+    label: Platform
+    description: Select a specific platform for this bug report, choose Any if it applies to all platforms
+    options:
+    - All
+    - Windows
+    - macOS
+    - NAS/Synology/QNAP
+    - Linux/Docker
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Environment
+    description: Please provide the details of the system NZBGet is running on. The more information you can provide, the better
+    placeholder: |
```

Comment:
Maybe separate sections ?
- Platform (windows/linux/nas)
- Architecture (arm / aarch64 / x86 / x86_64 etc)
- OS version
- Docker: yes/no

Reply:
Good suggestion, but let's do it later to keep it simple.